src/Frost.ts

```diff
@@ -341,4 +344,24 @@ export class Frost {
         throw e
       })
   }
+
+  getPublicKeys(cuid?: string): Promise<{ readonly publicKey: string }> {
```

Comment:
Is this endpoint available in Frost API already?

Can't find it in `master`

Reply:
@lautarodragan  Which comes first the client or the api? The api seems to rely on the client, so I presumed this came first.